lib/active_model_serializers/adapter/json_api.rb

```diff
@@ -294,7 +294,7 @@ def attributes_for(serializer, fields)
 
       # {http://jsonapi.org/format/#document-resource-objects Document Resource Objects}
       def resource_object_for(serializer)
-        resource_object = serializer.cache_check(self) do
+        resource_object = serializer.fetch(self) do
```

Comment:
(That's the whole diff, but will do) 

Any thoughts on changes this pr before it's merged?  

Next one I'm going to evolve a caching stage, I think